app/src/main/java/com/example/javadevelopersnairobi/javadevelopersnairobi/util/ConnectivityReceiver.java

```diff
@@ -0,0 +1,51 @@
+package com.example.javadevelopersnairobi.javadevelopersnairobi.util;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+public class ConnectivityReceiver extends BroadcastReceiver {
+
+	public static ConnectivityReceiverListener connectivityReceiverListener;
+
+	public ConnectivityReceiver(){
+		super();
+	}
+
+	@Override
+	public void onReceive(Context context, Intent intent){
+		ConnectivityManager connectivityManager = (ConnectivityManager) context
```

Comment:
There is a repetition here.

Reply:
Let me fix it thank you for the feedback